winui/DataGrid/drag-and-drop.md

```diff
@@ -0,0 +1,216 @@
+* `DraggingRows` : Gets the records which contains the data associated while dragging the rows.
+* `Cancel` : Gets and sets a value indicating whether the event is Canceled or not. 
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.RowDragStarting += SfDataGrid_RowDragStarting;
+
+private void SfDataGrid_RowDragStarting(object sender, DataGridRowDragStartingEventArgs e)
+{
+    
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Row Drag over event
+
+`RowDragOver` event occurs continuously while record is dragged within the target SfDataGrid. The `DataGridRowDragOverEventArgs` has the following members, which provide information for the `RowDragOver` event.
+* `Data` : Gets a data object that contains the data associated while dragging the rows. 
+* `DraggingRows` : Gets the records which contains the data associated while dragging the rows.
+* `DropPosition` : Gets a value indicating the drop position which is based on dropped location 
+* `ShowDragUI` : Gets or sets the visibility of Dragging UI.  
+* `TargetIndex` : Gets a value indicating the target index which is going to drop.
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.RowDragOver += SfDataGrid_RowDragOver;
+
+private void SfDataGrid_RowDragOver(object sender, DataGridRowDragOverEventArgs e)
+{
+
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Row Dropping event
+
+`RowDropping` event occurs when a record is dropping within the target SfDataGrid. The `DataGridRowDroppingEventArgs` has the following members, which provide information for the `RowDropping` event.
+* `Cancel` : Gets and sets a value indicating whether the event is Canceled or not. 
+* `Data` : Gets a data object that contains the data associated while dragging the rows. 
+* `DraggingRows` : Gets the records which contains the data associated while dragging the rows. 
+* `DropPosition` : Gets a value indicating the drop position which is based on dropped location 
+* `TargetIndex` : Gets a value indicating the target record which is going to drop.
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.RowDropping += SfDataGrid_RowDropping;
+
+private void SfDataGrid_RowDropping(object sender, DataGridRowDroppingEventArgs e)
+{
+
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Row Dropped event
+
+`RowDropped` event occurs when a record is dropping within the target SfDataGrid. The `DataGridRowDroppedEventArgs` has the following members, which provide information for the `Dropping` event.
+* `Data` : Gets a data object that contains the data associated while dragging the rows. 
+* `DraggingRows` : Gets the records which contains the data associated while dragging the rows.
+* `DropPosition` : Gets a value indicating the drop position which is based on dropped location 
+* `TargetIndex` : Gets a value indicating the target record which is going to drop.
```

Comment:
Need to modify the TargetIndex description.

Reply:
Review concerns addressed